tools/clang/test/CodeGenSPIRV/cf.discard.hlsl

```diff
@@ -1,7 +1,9 @@
 // Run: %dxc -T ps_6_0 -E main
```

Comment:
Why don't we add a test that uses `discard;` in a vertex shader and check it with the expected error?

Reply:
Done